cmsdb/campaigns/run2_2016_nano_uhh_v12/hh2bbtautau.py

```diff
@@ -0,0 +1,1278 @@
+    ],
+    n_files=1,
+    n_events=184000,
+)
+
+cpn.add_dataset(
+    name="hh_ggf_bbtautau_node8_madgraph",
+    id=14373531,
+    processes=[procs.hh_ggf_bbtautau_node8],
+    keys=[
+        "/GluGluToHHTo2B2Tau_TuneCP5_PSWeights_node_8_13TeV-madgraph-pythia8/RunIISummer20UL16NanoAODv12-106X_mcRun2_asymptotic_v17-v2/NANOAODSIM",  # noqa
+    ],
+    n_files=1,
+    n_events=184000,
+)
+
+cpn.add_dataset(
+    name="hh_ggf_bbtautau_node9_madgraph",
+    id=14373753,
+    processes=[procs.hh_ggf_bbtautau_node9],
+    keys=[
+        "/GluGluToHHTo2B2Tau_TuneCP5_PSWeights_node_9_13TeV-madgraph-pythia8/RunIISummer20UL16NanoAODv12-106X_mcRun2_asymptotic_v17-v2/NANOAODSIM",  # noqa
+    ],
+    n_files=1,
+    n_events=184000,
+)
+
+cpn.add_dataset(
+    name="hh_ggf_bbtautau_node10_madgraph",
+    id=14373533,
+    processes=[procs.hh_ggf_bbtautau_node10],
+    keys=[
+        "/GluGluToHHTo2B2Tau_TuneCP5_PSWeights_node_10_13TeV-madgraph-pythia8/RunIISummer20UL16NanoAODv12-106X_mcRun2_asymptotic_v17-v2/NANOAODSIM",  # noqa
+    ],
+    n_files=1,
+    n_events=184000,
+)
+
+cpn.add_dataset(
+    name="hh_ggf_bbtautau_node11_madgraph",
+    id=14374873,
+    processes=[procs.hh_ggf_bbtautau_node11],
+    keys=[
+        "/GluGluToHHTo2B2Tau_TuneCP5_PSWeights_node_11_13TeV-madgraph-pythia8/RunIISummer20UL16NanoAODv12-106X_mcRun2_asymptotic_v17-v1/NANOAODSIM",  # noqa
+    ],
+    n_files=1,
+    n_events=184000,
+)
+
+cpn.add_dataset(
+    name="hh_ggf_bbtautau_node12_madgraph",
+    id=14374899,
+    processes=[procs.hh_ggf_bbtautau_node12],
+    keys=[
+        "/GluGluToHHTo2B2Tau_TuneCP5_PSWeights_node_12_13TeV-madgraph-pythia8/RunIISummer20UL16NanoAODv12-106X_mcRun2_asymptotic_v17-v1/NANOAODSIM",  # noqa
+    ],
+    n_files=1,
+    n_events=184000,
+)
+
+
+#
+# ggF -> radion -> HH
+#
+
+cpn.add_dataset(
+    name="radion_hh_ggf_bbtautau_m250_madgraph",
+    id=14302510,
+    processes=[procs.radion_hh_ggf_bbtautau_m250],
+    keys=[
+        "/GluGluToRadionToHHTo2B2Tau_M-250_TuneCP5_PSWeights_narrow_13TeV-madgraph-pythia8/RunIISummer20UL16NanoAODv12-106X_mcRun2_asymptotic_v17-v1/NANOAODSIM",  # noqa
+    ],
+    n_files=1,
+    n_events=184000,
+)
+
+cpn.add_dataset(
+    name="radion_hh_ggf_bbtautau_m260_madgraph",
+    id=14301575,
+    processes=[procs.radion_hh_ggf_bbtautau_m260],
+    keys=[
+        "/GluGluToRadionToHHTo2B2Tau_M-260_TuneCP5_PSWeights_narrow_13TeV-madgraph-pythia8/RunIISummer20UL16NanoAODv12-106X_mcRun2_asymptotic_v17-v2/NANOAODSIM",  # noqa
+    ],
+    n_files=1,
+    n_events=184000,
+)
+
+cpn.add_dataset(
+    name="radion_hh_ggf_bbtautau_m270_madgraph",
+    id=14300691,
+    processes=[procs.radion_hh_ggf_bbtautau_m270],
+    keys=[
+        "/GluGluToRadionToHHTo2B2Tau_M-270_TuneCP5_PSWeights_narrow_13TeV-madgraph-pythia8/RunIISummer20UL16NanoAODv12-106X_mcRun2_asymptotic_v17-v2/NANOAODSIM",  # noqa
+    ],
+    n_files=1,
+    n_events=184000,
```

Comment:
Just out of interest, do all these samples for different masses really have the exact same number of events?

Reply:
I just checked the json and root files for the GluGlu radion mass 260, 270, 280 and 300, the numbers correspond to what is in cmsdb, so I guess the answer is yes